src/javascripts/legacy_app.js

```diff
@@ -1,5 +1,9 @@
 import BaseApp from 'base_app';
 
+// remove these if not needed
```

Comment:
These are here just in case the app is already using them, but sure, we could add a mention to the migration doc.